wordpress-mobile/gutenberg-mobile
# Fix crash on list controls

This PR updates the Gutenberg ref to test a crash fix described on: https://github.com/WordPress/gutenberg/pull/19818:

The editor crashes when trying to press on the list block controls.

To test:
- Add a List Block.
- Press any of the controls (i.e: Change the list type to ordered).
- Check that it doesn't crash.
- Check that it works properly.

PR submission checklist:
